### PR TITLE
CM-105: Params Keyword in C#

### PR DESCRIPTION
**Pull request by bartosz-jarmuz**

### CODE REVIEW

File: csharp-basic-topics/ParamsKeyword/ParamsKeyword/CallParamsMethod.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        {
+            //capture the console output
+            using var writer = new StringWriter();
+            Console.SetOut(writer);
+
+            var shoppingList = new ShoppingList();
+            string[]? groceries = default;
+
+#pragma warning disable CS8604 // Possible null reference argument.
+            Assert.Throws<NullReferenceException>(()=>shoppingList.Add(groceries));
```
Comment (CodeMazeBlog) [inline csharp-basic-topics/ParamsKeyword/ParamsKeyword/CallParamsMethod.cs:78]: Formatting problem.

File: csharp-basic-topics/ParamsKeyword/ParamsKeyword/CallParamsMethod.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+using System.Collections.Generic;
+using System.IO;
+
+namespace ParamsKeyword
+{
+    [TestFixture]
+    public class CallParamsMethod
+    {
+        [Test]
+        public void CommaDelimitedListOfArgs_ReturnsValidResult()
```
Comment (CodeMazeBlog) [inline csharp-basic-topics/ParamsKeyword/ParamsKeyword/CallParamsMethod.cs:12]: Please use the naming conventions from the guidelines so we can have a consistent experience throughout the code base.

**Review by CodeMazeBlog — CHANGES_REQUESTED** [review]
Minor fixes needed

File: csharp-basic-topics/ParamsKeyword/ParamsKeyword/CallParamsMethod.cs (hunk 1 above)
Comment (bartosz-jarmuz) [reply csharp-basic-topics/ParamsKeyword/ParamsKeyword/CallParamsMethod.cs]: preprocessor directives are by convention not indented at all, but I have moved them

Comment (CodeMazeBlog) [reply csharp-basic-topics/ParamsKeyword/ParamsKeyword/CallParamsMethod.cs]: I've meant ()=> in the Assert, sorry for not being clear enough.

**Review by CodeMazeBlog — APPROVED** [review]

**Pull request merged**